src/ResultsActivity/elements/PayOptions/PayOptions.js

```diff
@@ -87,10 +88,16 @@ function PayOptions({ offenderDetails, updateOffenderAttendance, date, noPay, sh
             </Radio>
           )}
         {allowUpdate && (
-          <UpdateLink role="link" onClick={renderForm}>
-            <OtherMessage data-qa="other-message">{absentReason.name}</OtherMessage>
-          </UpdateLink>
+          <Fragment>
+            {noPay && <OtherMessage data-qa="other-message">{absentReason.name}</OtherMessage>}
+            {!noPay && (
+              <UpdateLink role="link" onClick={renderForm}>
+                <OtherMessage data-qa="other-message">{absentReason.name}</OtherMessage>
+              </UpdateLink>
+            )}
+          </Fragment>
         )}
+        {!allowUpdate && notRecorded && noPay && <OtherMessage data-qa="other-message">Not Recorded</OtherMessage>}
```

Comment:
Feel free to ignore this, but could we reduce this to a shouldShowNotRecorded?

Reply:
I think I might leave it to remind us how wrong this is :D